### ISSUE TITLE
3.3.0 AlleleCall reports missing files

### ISSUE THREAD

**Issue by karlkashofer** [B-UMMI/chewBBACA#186]
It seems AlleleCall is broken in 3.3.0 built inside your dockerfile:
`docker run -v $PWD:/analysis chewbbaca:3.3.0 bash -c "cd /analysis;chewBBACA.py AlleleCall -i Expt1_genomes -g schema/Pseudomonas --gl Pseudomonas_schema_calls/cgMLST/cgMLSTschema95.txt -o result_Expt1_with_infer --cpu 30 &> log"`

```chewBBACA version: 3.3.0
Authors: Rafael Mamede, Pedro Cerqueira, Mickael Silva, João Carriço, Mário Ramirez
Github: https://github.com/B-UMMI/chewBBACA
Documentation: https://chewbbaca.readthedocs.io/en/latest/index.html
Contacts: imm-bioinfo@medicina.ulisboa.pt

==========================
  chewBBACA - AlleleCall
==========================
Started at: 2023-09-11T10:51:15

Could not find some of the files provided in the input list. Please verify that you've provided the full paths to valid input files.
schema/Pseudomonas/GCA-000006765-PAO1-protein10
schema/Pseudomonas/GCA-000006765-PAO1-protein100
schema/Pseudomonas/GCA-000006765-PAO1-protein1014
schema/Pseudomonas/GCA-000006765-PAO1-protein1016
schema/Pseudomonas/GCA-000006765-PAO1-protein1018
schema/Pseudomonas/GCA-000006765-PAO1-protein1023
schema/Pseudomonas/GCA-000006765-PAO1-protein1024
schema/Pseudomonas/GCA-000006765-PAO1-protein1025
```
The same command works fine when i use 3.2.0 from conda.

**Comment by rfm-targa** [comment]
Hello @karlkashofer,

Thank you for reporting this issue. This is one of the few bugs we will fix before releasing v3.3.0. The process is failing because the loci identifiers in the file passed to the `--gl` parameter do not end with `.fasta`. It should work if you append `.fasta` to every line in the `cgMLSTschema95.txt` file.
Does it also fail when you do not use the `--gl` parameter?

Best regards,

Rafael

**Comment by rfm-targa** [comment]
Hello @karlkashofer,

The issue related to the `--gl` parameter should be fixed in the latest release, v3.3.0 (you can install it through [pip](https://pypi.org/project/chewBBACA/#description) or [conda](https://anaconda.org/bioconda/chewbbaca)). Closing this issue, but let us know if you try it and it is still a problem.

Kind regards,

Rafael
